### PR TITLE
Added a CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by phadnisvinay30**
please have a look and feel free to tell me about any changes or do it yourself

### CODE REVIEW

**Comment by Rovak** [comment]
there are some `java-tron` references that should be renamed to `scala-tron`

**Comment by phadnisvinay30** [comment]
working on it

**Comment by phadnisvinay30** [comment]
Fixed it !

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+email, or on [Gitter](https://gitter.im/tronprotocol/java-tron) with the owners of this repository before making a change. 
+
+Please note we have a code of conduct and you are requested to please follow it in all your interactions with the project.
+<br />
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Scala-tron follows [gitflow workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). Please open pull  requests to the develop branch. Once approved, we will close the pull request and merge into master branch.
+4. If your pull request is not accepted on the first try, don't be discouraged as it can be a possible oversight. Please explain your code as detailed as possible to make it easier for us to understand.
```
Comment (Napokue) [inline CONTRIBUTING.md]: This should be number 3

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+# Contributing
+scala-tron is an open source project.
+
+It is the work of contributors. We appreciate your help!
+
+Here are instructions to get you started. They are not perfect, so please let us know if anything feels wrong or incomplete.
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or on [Gitter](https://gitter.im/tronprotocol/java-tron) with the owners of this repository before making a change.
```
Comment (Napokue) [inline CONTRIBUTING.md]: Telegram is also used as communication tool, you could add that here too. This is the group chat https://t.me/joinchat/CP8XKBIjEc0RqzJdl61OlQ

Comment (Napokue) [reply CONTRIBUTING.md]: The gitter link is that of java-tron

File: CONTRIBUTING.md (hunk 1 above)
Comment (phadnisvinay30) [reply CONTRIBUTING.md]: working on it

**Comment by Napokue** [comment]
Looks good to me!

**Comment by Rovak** [comment]
LGTM! 👍

**Pull request merged**